var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -30,6 +32,7 @@ class Exago(CMakePackage, CudaPackage):
     variant('mpi', default=True, description='Enable/Disable MPI')
     variant('raja', default=False, description='Enable/Disable RAJA')
     variant('python', default=True, description='Enable/Disable Python bindings')
+    conflicts('+python', when='@:1.3.0', msg='Python bindings require ExaGO 1.3')
```

Comment:
I think this is a typo: as currently encoded, this means that `+python` is only allowed for `@1.3.1:` (i.e. versions that are strictly after `1.3.0`).

Reply:
That's correct - Python should only be allowed on or after 1.4.0, so the message should be updated.